examples/nuxt-openai/.gitignore

```diff
@@ -21,3 +22,4 @@ logs
 .env
 .env.*
 !.env.example
+.dev.vars
```

Comment:
Since we are at it, please add `.data` too (will be used in the future for nuxt cache/data)